github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+|                    | area-linux               |                  |                 |
+|                    | area-management          |                  |                 |
+|                    | area-operations          |                  |                 |
+|                    | area-reliability         |                  |                 |
+|                    | area-security            |                  |                 |
+|                    | area-setup               |                  |                 |
+|                    | area-sfrp                |                  |                 |
+|                    | area-standaloneinstaller |                  |                 |
+|                    | area-store               |                  |                 |
+|                    | area-test                |                  |                 |
+|                    | area-testability         |                  |                 |
+|                    | area-transport           |                  |                 |
+|                    | area-needs-triage        |                  |                 |
+|                    | area-try-service-fabric  |                  |                 |
+|                    | varea-windows-server     |                  |                 |
+|                    |                          |                  |                 |
+
+Project - NA for now
+--------------------------------------------------------------------------
+
+| ***xxxx*** | ***xxx*** |
+|------------|-----------|
+| xxx        | xxx       |
+
+
+> **No labels for this area at this time.**
+
+Milestones
+----------------------------------------------------------------
+
+| ***Release (Iteration path)***                                                            |
+|-------------------------------------------------------------------------------------------|
+| 6.3, 6.4, etc. (define specifically once we get a build number at end of release)         
+                                                                                            
+ 6.3 CU1, 6.4 CU1, etc. (define specifically once we a get build number at end of release)  
+                                                                                            
+ > Example 6.3.xxx.xx                                                                       
+ >                                                                                          
+ > All work items update automatically once the milestone is updated. (= minimal effort)    
+                                                                                            
+ There is not a vNext or backlog release.                                                   
+                                                                                            
+ > No Milestone = backlog                                                                   |
+
+Happy Path By issue
```

Comment:
I don't think we should try to enumerate all the possible flows.  I would advocate instead for describing clearly what the labels mean, and perhaps give some specific examples of situations in which they might be used.  Even though these paths very well might be how they are used **sometimes** describing them in this way reads more like a process we expect to be followed as opposed to some guidelines.  

In partiular saying `needs-more-info (optional)` implies that the other "state/phases" are **not** optional.  I think fundamentally this reads as rigid states with transitions as opposed to variables you can mix and match to describe the state of an issue.  I think this could be reworded to instead focus on what the labels mean.